src/apps/Altinn.AccessManagement/src/Altinn.AccessManagement.Core/Errors/ValidationErrors.cs

```diff
@@ -76,4 +76,130 @@ private static readonly ValidationErrorDescriptorFactory _factory
+        = _factory.Create(25, $"Unknown consent metaddata.");
+
+    /// <summary>
+    /// Gets a validation error descriptor
+    /// </summary>
+    public static ValidationErrorDescriptor MissingMetadataValue { get; }
+        = _factory.Create(26, $"Missing value for metadata");
+
+    /// <summary>
+    /// Gets a validation error descriptor
+    /// </summary>
+    public static ValidationErrorDescriptor MissingMetadata { get; }
+        = _factory.Create(27, $"Missing required metadata for consentright");
+
+    /// <summary>
+    /// Gets a validation error descriptor
+    /// </summary>
+    public static ValidationErrorDescriptor MissingAction { get; }
+        = _factory.Create(28, $"Missing required actions for consent request");
+
+    /// <summary>
+    /// Gets a validation error descriptor
+    /// </summary>
+    public static ValidationErrorDescriptor MissMatchConsentParty { get; }
+    = _factory.Create(29, $"The consented party does not match the party requested");
+
+    /// <summary>
+    /// Gets a validation error descriptor
+    /// </summary>
+    public static ValidationErrorDescriptor ConsentNotFound { get; }
+    = _factory.Create(30, $"Incorrect consentId or wrong consent receiver");
+
+    /// <summary>
+    /// Gets a validation error descriptor
+    /// </summary>
+    public static ValidationErrorDescriptor ConsentExpired { get; }
+    = _factory.Create(31, $"Consent is expired");
+
+    /// <summary>
+    /// Gets a validation error descriptor
+    /// </summary>
+    public static ValidationErrorDescriptor ConsentNotAccepted { get; }
+    = _factory.Create(32, $"Consent is not accepted");
+
+    /// <summary>
+    /// Gets a validation error descriptor
+    /// </summary>
+    public static ValidationErrorDescriptor ConsentRevoked { get; }
+    = _factory.Create(33, $"Consent is revoked");
+
+    /// <summary>
+    /// Gets a validation error descriptor
+    /// </summary>
+    public static ValidationErrorDescriptor ConsentCantBeAccepted { get; }
+    = _factory.Create(34, $"Consent cant be accepted. Wrong status");
+
+    /// <summary>
+    /// Gets a validation error descriptor
+    /// </summary>
+    public static ValidationErrorDescriptor ConsentCantBeRevoked { get; }
+    = _factory.Create(35, $"Consent cant be revoked. Wrong status");
+
+    /// <summary>
+    /// Gets a validation error descriptor
+    /// </summary>
+    public static ValidationErrorDescriptor ConsentCantBeRejected { get; }
+    = _factory.Create(36, $"Consent cant be rejected. Wrong status");
+
+    /// <summary>
+    /// Gets a validation error descriptor
+    /// </summary>
+    public static ValidationErrorDescriptor InvalidResourceCombination { get; }
+    = _factory.Create(37, $"These resources cannot be combined.");
+
+    /// <summary>
+    /// Gets a validation error descriptor
+    /// </summary>
+    public static ValidationErrorDescriptor MissingContext { get; }
```

Comment:
This is probably not a validation error, but rather a full problem. - either that, or it should just use `Required` from line 18.